### ISSUE TITLE
Challenges with colon in title

### ISSUE THREAD

**Issue by rgaudin** [openzim/freecodecamp#6]
Just saw another glitch ; in `python-for-everybody`, a number of challenges like `networking-text-processing` have titles that include a colon `:` which breaks the YAML parsing.

See `/fcc/curriculum/english/scientific-computing-with-python/python-for-everybody/networking-text-processing.md`:

```yaml
---
id: 5e7b9f0c0b6c005b0e76f074
title: 'Networking: Text Processing'
challengeType: 11
videoId: Pv_pJgVu8WI
bilibiliIds:
  aid: 804442498
  bvid: BV16y4y1j7WW
  cid: 377329124
dashedName: networking-text-processing
---
```

The rendered title is `'Networking`

<img width="568" alt="Screenshot 2023-07-04 at 09 44 45" src="https://github.com/openzim/freecodecamp/assets/57929/7303f7c5-ce78-4668-b7ee-e23b2829ae9b">

_Originally posted by @rgaudin in https://github.com/openzim/freecodecamp/issues/5#issuecomment-1619706243_

**Comment by mdp** [comment]
This isn't an issue with Python, but happening on the render side. This needs to be solved in conjuction with #8

**Comment by benoit74** [comment]
All project euler challenges have a semi-colon, to be fixed soon.
